github/issues_test.go

```diff
@@ -275,3 +275,14 @@ func TestIssuesService_Unlock(t *testing.T) {
 		t.Errorf("Issues.Unlock returned error: %v", err)
 	}
 }
+
+func TestIsPullRequest(t *testing.T) {
+	i := new(Issue)
+	if i.IsPullRequest() == true {
```

Comment:
> I often refer to tests as an example of how to use a function

Tests usually do weird things to test functionality and not optimize for readability. Examples are for that.

The `== true` part to me is basically a `== dontWant`, with `true` inlined. We could also be more explicit and write the more verbose version thereof:

```Go
if got, want := i.IsPullRequest(), true; got != want {
```

Either way is fine with me.

I don't think we need to worry about `if i.IsPullRequest() == notWant {` being misinterpreted as the usage of this method.